docs/Installation/Installation.md

```diff
@@ -58,7 +58,9 @@ all of these dependencies.
 #### Required:
 
 * [GCC](https://gcc.gnu.org/) 9.1 or later,
-[Clang](https://clang.llvm.org/) 11.0 or later, or AppleClang 11.0.0 or later
+[Clang](https://clang.llvm.org/) 12.0 or later (see
```

Comment:
There's no test for clang 12. Do you want to add one or bump to clang 13?

Reply:
We can bump to 13, I think. Wheeler has clang-13 installed.

Reply:
Ocean also has llvm 13 installed so all good there

Reply:
CaltechHPC has only clang 12, but I don't care

Reply:
Sounds good. And we can ask them to update. That should be doable.

Reply:
I'll open a ticket with them.